docker-compose.multi-storage.yml

```
@@ -0,0 +1,36 @@
+version: '3.4'
+services:
+  colossus-2:
+    image: joystream/colossus:latest
+    restart: on-failure
+    build:
+      context: .
+      dockerfile: colossus.Dockerfile
+    depends_on:
+      - graphql-server
+    volumes:
+      - /data
+      - /keystore
+    ports:
+      - '127.0.0.1:3335:${COLOSSUS_PORT_2}'
```

I would have expected to only need to bind on host to a different port, but in container port is unchanged.
```suggestion
      - '127.0.0.1:${COLOSSUS_PORT_2}:3334'
```